src/cellars/adaptors/aave_v2.rs

```diff
@@ -0,0 +1,315 @@
+use abscissa_core::tracing::log::debug;
+use ethers::{abi::AbiEncode, types::Bytes};
+
+use crate::{
+    abi::adaptors::{
+        aave_v2_a_token_adaptor_v1::AaveATokenAdaptorV1Calls as AbiAaveATokenAdaptorV1Calls,
```

Comment:
Is support for v1 of the Aave V2 adaptor a leftover from the original Aave cellar?

Reply:
We decided to only remove support for adaptors when they are deprecated because of a bug or security concern. afaik v1 of this is still valid.